rules_java_gapic/resources/gradle/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.3-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-bin.zip
```

Comment:
Is this because Gradle 8.4 is the minimum version that supports Java 21?

Reply:
yes, as per https://docs.gradle.org/current/userguide/compatibility.html